seo_app/allurl.py

```diff
@@ -0,0 +1,53 @@
+from bs4 import BeautifulSoup as sp
+import requests
+
+url = 'https://www.geeksforgeeks.org/'
```

Comment:
This project is not for a specific URL based scraping, but for generic scraping of webpages.